sim/src/mechanics/parking.rs

```diff
@@ -579,9 +589,12 @@ impl ParkingSim for NormalParkingSimState {
             for turn in map.get_turns_for(current, PathConstraints::Car) {
                 if !backrefs.contains_key(&turn.id.dst) {
                     let dist_this_step = turn.geom.length() + map.get_l(current).length();
+                    // When vehicles search away from the first lane for a spot, don't all go in
+                    // the same direction! Do this by jittering which turn they explore.
+                    let jitter = rng.gen_range(0.1..0.9);
```

Comment:
So at *worst* they'll consider a route to be 10% of its true length - meaning at worst, they'll go 10x farther than necessary. Right?

Might be worth running a smoke test to verify that overall travel times aren't suddenly wayyyy worse across the board, in which case maybe we'll need to tune this.

Reply:
Your logic sounds right (and I'm adding it as a comment.)

![timediff](https://user-images.githubusercontent.com/1664407/106981874-bbf16400-6717-11eb-90b0-d38bbb5870ab.png)
Luckily we have a way of seeing if a change makes times overall change much. For all 3 working maps, it looked about like this -- some people win a bit, some people lose.

Reply:
excellent!